joomla/joomla-cms
# [4.0] Prevent two messages displayed for creating override

Pull Request for Issue #31894.

### Summary of Changes
This small PR fixes the issue https://github.com/joomla/joomla-cms/issues/31894. It prevents two messages displayed when override is created in the template


### Testing Instructions
1. See https://github.com/joomla/joomla-cms/issues/31894 , confirm the issue
2. Apply patch, try again. Now only one message is displayed.
